ExperimentalFeatures.json

```diff
@@ -0,0 +1,12 @@
+[
+  "PSAnsiRenderingFileInfo",
+  "PSCleanBlock",
+  "PSCommandNotFoundSuggestion",
+  "PSLoadAssemblyFromNativeCode",
+  "PSNativeCommandArgumentPassing",
+  "PSNativeCommandErrorActionPreference",
+  "PSNativePSPathResolution",
+  "PSRemotingSSHTransportErrorHandling",
+  "PSSubsystemPluginModel",
+  "PSAMSIMethodInvocationLogging"
+]
```

Comment:
If there is no automation to keep this in sync with code, it's probably will be out of sync sooner or later.

Reply:
Added automation via GitHub Actions in PR #16838